From fdc3d79b7b68d3c02ceb79f45c2cf03d4ed0f529 Mon Sep 17 00:00:00 2001
From: b883 <b883@borqs.com>
Date: Fri, 13 Nov 2015 08:55:47 +0800
Subject: [PATCH 3/4] [BT] Reorder GPIO idx

The GPIO idx used in gpio_get_index() shall
be reordered in order to avoid a current leak

Change-Id: Ieee91b82447eb3a8a5db8ae91476723678c781d2
Borig-Change-Id: Iffbfc9d39a236600e061a5f59e1bea2c3ea1b404
Reviewed-on: http://192.168.1.79/gerrit/14830
Reviewed-by: b225 <b225@borqs.com>
Reviewed-on: https://android.intel.com:443/436134
Signed-off-by: Marco Chiotti <marcox.chiotti@intel.com>
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-20208
Reviewed-on: https://android.intel.com:443/471654
---
 drivers/misc/bt_lpm/bt_lpm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/misc/bt_lpm/bt_lpm.c b/drivers/misc/bt_lpm/bt_lpm.c
index 9a6ba14..7205541 100644
--- a/drivers/misc/bt_lpm/bt_lpm.c
+++ b/drivers/misc/bt_lpm/bt_lpm.c
@@ -29,8 +29,8 @@
 #include <linux/serial_hsu.h>
 
 enum {
-	gpio_wake_acpi_idx,
 	gpio_enable_bt_acpi_idx,
+	gpio_wake_acpi_idx,
 	host_wake_acpi_idx
 };
 static struct gpio_desc *bt_lpm_gpiod;
-- 
1.9.1

